src/main/java/com/example/Game.java

```diff
@@ -3,88 +3,30 @@
-        int size;
-        while (true)
-        {
-            try{
-                size = scanner.nextInt();
-                scanner.nextLine();
-                if(size>=3)
-                {
-                    break;
-                }
-                else {
-                    System.out.println("Grid size must be 3 or larger than 3");
-                }
-            }
-            catch (InputMismatchException e)
-            {
-                System.out.println("Invalid input! Please enter valid number");
-            }
-        }
-        board = new Board(size);
-
-        System.out.println("So, tell me what's your name?");
-        String playerName;
-        while (true)
-        {
-            playerName = scanner.nextLine();
-            if(!playerName.isEmpty())
-            {
-                break;
-            }
-            else {
-                System.out.println("Name cannot be empty. Please enter a valid name.");
-            }
-        }
+        System.out.println("Bored! Let's play a Game!! \uD83C\uDFB2");
 
+        int size = getGridSize();
+        board = new Board(size);
 
-        char playerSymbol;
-        while (true) {
-            System.out.println("Choose your symbol (X or O):");
-            playerSymbol = Character.toUpperCase(scanner.next().charAt(0));
-            if (playerSymbol == 'X' || playerSymbol == 'O') {
-                break;
-            } else {
-                System.out.println("Invalid symbol! Please choose X or O.");
-            }
-        }
-        player1 = new HumanPlayer(playerName, playerSymbol);
-
-        int difficultyLevel;
-        while (true) {
-            System.out.println("Choose difficulty level: 1 for EASY and 2 for HARD");
-            try {
-                difficultyLevel = scanner.nextInt();
-                scanner.nextLine();
-                if (difficultyLevel == 1 || difficultyLevel == 2) {
-                    break;
-                }
-                else
-                {
-                    System.out.println("Please enter 1 for EASY or 2 for HARD.");
-                }
-            } catch (InputMismatchException e) {
-                System.out.println("Invalid input. Please enter a valid number.");
-                scanner.next();
-            }
-        }
+        // Create player1 by using PlayerFactory and creating instance of HumanPlayerFactory
+        PlayerFactory humanFactory = new HumanPlayerFactory();
+        player1 = humanFactory.createPlayer();
 
-        char cpuSymbol = (playerSymbol == 'X') ? 'O' : 'X';
-        MoveStrategy strategy = (difficultyLevel == 1) ? new RandomMoveStrategy() : new SmartMoveStrategy();
-        player2 = new CpuPlayer(strategy,cpuSymbol);
+        // Create player2 by using PlayerFactory and creating instance of CpuPlayerFactory
+        char cpuSymbol = (player1.getSymbol() == 'X') ? 'O' : 'X';
+        PlayerFactory cpuFactory = new CpuPlayerFactory(cpuSymbol);
```

Comment:
Computer factory should not depend on symbol,
Have an extra method for setting symbol